### PR TITLE
Bashing doc bugs.

### PR DESCRIPTION
**Pull request by MichaelNorman**
### Description of Change ###

Minor doc updates in response to bugs. Passed local pre-commit linting.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Consolidate commits as makes sense

### CODE REVIEW

File: docs/Xamarin.Forms.Core/Xamarin.Forms/ListViewCachingStrategy.xml [hunk 1]
```diff
@@ -25,7 +25,10 @@
...
+        <block type="note">When devolopers specify <see cref="F:Xamarin.Forms.ListViewCachingStrategy.RecycleElement" />, <c>OnElementChanged</c> events are not raised when cells are recycled. Instead,  the cell is retained and its property values change when the binding context is updated to that of an available cell, <c>OnElementPropertyChanged</c> events are raised. Application developers should remember to listen for the correct events, and should note that their renderers will need to be updated if the default behavior changes to <see cref="F:Xamarin.Forms.ListViewCachingStrategy.RecycleElement" /> in a future release.</block>
```
Comment (Therzok) [inline docs/Xamarin.Forms.Core/Xamarin.Forms/ListViewCachingStrategy.xml]: This should be `When developers...`

File: docs/Xamarin.Forms.Core/Xamarin.Forms/ListView.xml [hunk 2]
```diff
@@ -237,7 +237,7 @@ class ListViewDemoPage : ContentPage
...
+            <block type="note">When developers specify <see cref="F:Xamarin.Forms.ListViewCachingStrategy.RecycleElement" />, <c>OnElementChanged</c> events are not raised when cells are recycled. Instead,  the cell is retained and its property values change when the binding context is updated to that of an available cell, <c>OnElementPropertyChanged</c> events are raised. Application developers should remember to listen for the correct events, and should note that their renderers will need to be updated if the default behavior changes to <see cref="F:Xamarin.Forms.ListViewCachingStrategy.RecycleElement" /> in a future release. </block>
```
Comment (hartez) [inline docs/Xamarin.Forms.Core/Xamarin.Forms/ListView.xml]: > Instead,  the cell
Looks like there's an extra space here.

Comment (MichaelNorman) [inline docs/Xamarin.Forms.Core/Xamarin.Forms/ListView.xml]: True enough, but HTML won't render it, and changing it means another round of unbreaking the broken tests. I say we leave it? Do you agree, @hartez?

Comment (hartez) [inline docs/Xamarin.Forms.Core/Xamarin.Forms/ListView.xml]: 👍

**Pull request merged**